src/utils/utils.tsx

```diff
@@ -60,3 +64,19 @@ export const getMapboxLatitudeOffset = (map: mapboxgl.Map | undefined) => {
 export const withLocaleUrl = (path: string) => {
   return `/${language}/${path}`;
 }
+
+export const translateAntibodyTargets = (antibodyTarget: string | string[]) => {
+  if (typeof antibodyTarget == "string") {
+    return Translate("AntibodyTargetOptions", [antibodyTarget.replace(/\s/g, "")])
+  }
+  else {
+    let i = 0
+    let returnString = ""
+    while(antibodyTarget[i]) {

```

Comment:
This would error out once i exceeds the length of the array no?

Might be cleaner to do something like this:

```
initialValue = ""
returnString =antibodyTarget.reduce(
  (previousValue, currentValue) => previousValue + Translate("AntibodyTargetOptions", [currentValue.replace(/\s/g, "")]) + ", ",
 initialValue
);
// To get rid of the trailing ", "
returnString = returnString.slice(0, -2)
```